### PR TITLE
Upload test apk's as a separate artifact

### PR DESCRIPTION
**Pull request by yigit**
This PR fixes a problem where we stopped distributing test apks hence
stopped running FTL tests (aosp/1751630).

We will now re-use what aosp does to reduce chances of this breaking
(should probably add some failsafe, this PR does not do it).

For this to take affect, we also need to update androidx-ci-action to
look for the new artifact name.

Bug: n/a
Test: CI

### CODE REVIEW

**Comment by yigit** [comment]
just trying on prod :) to see if the artifacts will show up properly

**Comment by dlam** [comment]
Oh, I think you don't need `\` if we have an open `"` already.

File: .github/workflows/presubmit.yml [hunk 1]
```diff
@@ -233,4 +233,5 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: artifacts_${{ env.artifact-id }}
-          path: ~/dist
+          path: |
```
Comment (dlam) [inline .github/workflows/presubmit.yml]: Can we undo this change?

**Review by dlam — APPROVED** [review]
